tools/wltests/platforms/xz2_android/definitions

```diff
@@ -0,0 +1,52 @@
+################################################################################
+# Build definitions for Hikey960
+################################################################################
+
+PLATFORM_NAME="xz2"
```

Comment:
What about akari ? And also, I think the folder should be called akari_v4.9, to be consistent with other platforms.

Reply:
Actually I willingly went against the naming convention of the other platforms because specifying the kernel version doesn't bring a lot of value TBH (I use hikey960_v4.4 to build a v4.14 kernel...)

Although you're right on the akari thing.

Reply:
I think we sort of agreed that the kernel version is meant to be the lowest supported (/tested) kernel, even though I agree this isn't bringing a lot of useful information in practice ...

Reply:
Fair enough, I'll add it :)